metrics/src/main/java/ai/vespa/metrics/Unit.java

```diff
@@ -12,6 +12,7 @@ public enum Unit {
     CONNECTION(BaseUnit.CONNECTION, "A link used for communication between a client and a server"),
     DOCUMENT(BaseUnit.DOCUMENT, "Vespa document, a collection of fields defined in a schema file"),
     DOCUMENTID(BaseUnit.DOCUMENTID, "A unique document identifier"),
+    DOLLAR_PER_HOUR(BaseUnit.MAGNITUDE, "$/hr"),
```

Comment:
If we need documentation for this metric, please add BaseUnits 'dollar' and 'hour' and declare the unit with per_unit hour.

Reply:
Done. It would be nice with some explanation of what baseunit is for.